config/config_params.go

```diff
@@ -508,6 +517,7 @@ func loadParams() {
 		case "millis":
 			param = &MillisParam{}
 		case "iface-list":
+			// TODO: What should we allow here given that it's now a regexp?
```

Comment:
I guess anything for which calling `InterfaceExcludes` doesn't panic?

Reply:
Fair point. This is now dictated by whatever we're allowing in terms of regexp patterns as values in the list. 

I'm going to separate this out into a new switch case, since it's reused right now between two separate parameters: 
- `InterfacePrefix`
- `InterfaceExclude`

And it's only the second one `InterfaceExclude` that we want to allow regexp for. 

Will also need to update the error message from: 
- `invalid Linux interface name` 

To something that calls out regexp patterns: 
- e.g. `invalid Linux interface name or regular expression pattern`

cc @caseydavenport